```
/home/runner/work/kvrocks/kvrocks/tests/cppunit/types/timeseries_test.cc:124:34: error: code should be clang-formatted [-Wclang-format-violations]
  EXPECT_TRUE(query_res.empty());
```
  
timeseries_test.cc not formarted. Could you fix it?

--- a/src/commands/cmd_timeseries.cc
+++ b/src/commands/cmd_timeseries.cc
@@ -315,7 +315,6 @@ class CommandTSCreateBase : public KeywordCommandBase {
     return Status::OK();
   }
 
-  //  private:
   TSCreateOption create_option_;
 };
 

--- a/tests/cppunit/types/timeseries_test.cc
+++ b/tests/cppunit/types/timeseries_test.cc
@@ -123,7 +123,6 @@ TEST_F(TimeSeriesTest, Alter) {
   EXPECT_TRUE(ts_db_->QueryIndex(*ctx_, filters, &query_res).ok());
   EXPECT_TRUE(query_res.empty());
 
-
   key_ = "pavni";
   auto s = ts_db_->Alter(*ctx_, key_, option, 1);
   EXPECT_FALSE(s.ok());

